query/queries_and_views.ipynb

```diff
@@ -854,6 +859,2063 @@
+       "      <td>410620009</td>\n",
+       "      <td>10392</td>\n",
+       "      <td>3445</td>\n",
+       "      <td>1064</td>\n",
+       "      <td>1063</td>\n",
+       "      <td>1963-12-17T03:03:25+00:00</td>\n",
+       "      <td>1963-12-17T03:18:25+00:00</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>10571</td>\n",
+       "      <td>finished</td>\n",
+       "      <td>162673000</td>\n",
+       "      <td>10392</td>\n",
+       "      <td>3445</td>\n",
+       "      <td>1064</td>\n",
+       "      <td>1063</td>\n",
+       "      <td>1980-01-22T03:03:25+00:00</td>\n",
+       "      <td>1980-01-22T03:18:25+00:00</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>11846</td>\n",
+       "      <td>finished</td>\n",
+       "      <td>162673000</td>\n",
+       "      <td>10669</td>\n",
+       "      <td>4409</td>\n",
+       "      <td>2028</td>\n",
+       "      <td>2027</td>\n",
+       "      <td>2017-12-29T20:07:11+00:00</td>\n",
+       "      <td>2017-12-29T20:22:11+00:00</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  enc_id    status       code patient_id practitioner_id location_id  \\\n",
+       "0  10357  finished  162673000       9040            2791         410   \n",
+       "1  10372  finished  162673000       9040            2791         410   \n",
+       "2  10523  finished  410620009      10392            3445        1064   \n",
+       "3  10571  finished  162673000      10392            3445        1064   \n",
+       "4  11846  finished  162673000      10669            4409        2028   \n",
+       "\n",
+       "  service_org_id                      start                        end  \n",
+       "0            409  2019-01-03T03:22:24+00:00  2019-01-03T03:37:24+00:00  \n",
+       "1            409  2021-01-14T03:22:24+00:00  2021-01-14T03:37:24+00:00  \n",
+       "2           1063  1963-12-17T03:03:25+00:00  1963-12-17T03:18:25+00:00  \n",
+       "3           1063  1980-01-22T03:03:25+00:00  1980-01-22T03:18:25+00:00  \n",
+       "4           2027  2017-12-29T20:07:11+00:00  2017-12-29T20:22:11+00:00  "
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "with query_engine.connect() as con:\n",
+    "    con.execute(\"\"\"\n",
+    "      CREATE OR REPLACE VIEW flat_encounter AS\n",
+    "      SELECT E.id AS enc_id, E.status, ETC.code, E.subject.PatientId AS patient_id,\n",
+    "        EP.individual.practitionerId AS practitioner_id,\n",
+    "        EL.location.locationId AS location_id,\n",
+    "        E.serviceProvider.organizationId AS service_org_id,\n",
+    "        E.period.start, E.period.end\n",
+    "      FROM Encounter AS E LATERAL VIEW OUTER explode(type) AS ET\n",
```

Comment:
For encounter, suggest adding:
*  encounter_type
* episodeOfCare -> this would require a new episodeOfCare view